guides/source/ja/engines.md

```diff
@@ -1006,3 +1006,114 @@ module MyEngine
   end
 end
 ```
+
+ロードフック上のActive Support
+----------------------------
+
+Active SupportはRuby言語の拡張、ユーティリティとそれ以外の巡回用ユーティリティを提供する責任を持つRuby on Railsのコンポーネントです。
+
+Railsコードはアプリケーション読み込みの時点でよく参照されます。Railsはこれらのフレームワークの読み込み順番に責任があのので、`ActiveRecord::Base`みたいなフレームワークを読み込む時にはアプリケーションがRailsと交わした暗黙的な約束を違反してしまう可能性があります。延いては`ActiveRecord::Base`みたいなコードをアプリケーションの起動時に読み込むと全体のフレームワークを読み込むことになり、起動時間を長くなる可能性があり、アプリケーションの起動時の読み込み順番での衝突が起きる可能性もあります。
+
+ロードフックはRailsの読み込み契約を乱すことなく初期化プロセスに介入することができるAPIです。それだけではなく、起動時間の問題を軽減したり、衝突問題を回避できるようにします。
```

Comment:
On Load hooks are the API that allow you to hook into this initialization process without violating the load contract with Rails. This will also mitigate boot performance degradation and avoid conflicts.